QF-867 Added 2 more exceptions to proper case rules

ChrisJWoodcock opened the pull request.

SawNawDfE commented on src/SFA.DAS.Assessor.Functions.UnitTests/Print/PrintExtensionFunction/StringNameCaseExtensionTests.cs:
```diff
@@ -5,18 +5,30 @@ namespace SFA.DAS.Assessor.Functions.UnitTests.Print.PrintExtensionFunction
+            var familyNameToProperCase = familyName.ProperCase(true);
+
+            // Assert
+            Assert.AreEqual(expectedToProperCase, upperCaseToProperCase, "Failed to proper case an upper case family name");
+            Assert.AreEqual(expectedToProperCase, lowerCaseToProperCase, "Failed to proper case an lower case family name");
+            Assert.AreEqual(expectedToProperCase, familyNameToProperCase, "Failed to proper case a mixed case family name");
```
If either of the first two asserts fail, the test run will stop without checking the remaining asserts. This will result in incorrect or incomplete test reports. 

We can prevent this by wrapping multiple assert statements inside an `Assert.Multiple()`:

```suggestion
            Assert.Multiple(() =>
            {
                Assert.AreEqual(expectedToProperCase, upperCaseToProperCase, "Failed to proper case an upper case family name");
                Assert.AreEqual(expectedToProperCase, lowerCaseToProperCase, "Failed to proper case an lower case family name");
                Assert.AreEqual(expectedToProperCase, familyNameToProperCase, "Failed to proper case a mixed case family name");
            });
```

SawNawDfE replied:
Looks good now, I do wonder if these should be 3 separate tests though

SawNawDfE commented:
> Please remember to check any packages used by this application to ensure they are up to date @ChrisJWoodcock. cc/ @ecarroll95

@ChrisJWoodcock Regarding packages, the NUnit3TestAdapter package is 1 major version out of date, shall we update it?

The MSTest packages are 2 major versions out of date, but it doesn't look like they're being used at all.

ChrisJWoodcock commented:
> > Please remember to check any packages used by this application to ensure they are up to date @ChrisJWoodcock. cc/ @ecarroll95
> 
> @ChrisJWoodcock Regarding packages, the NUnit3TestAdapter package is 1 major version out of date, shall we update it?
> 
> The MSTest packages are 2 major versions out of date, but it doesn't look like they're being used at all.

I've updated the NUnit packages and removed the MSTest packages - the other other packages will have to wait until we do something else which would have extensive manual testing as part of it - as this is only a very minor bug we can't do that testing in this.
